src/main/java/org/gridsuite/study/server/dto/sensianalysis/SensitivityAnalysisHvdcParameterInfos.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.dto.sensianalysis;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.study.server.dto.SensitivityAnalysisInputData;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+public class SensitivityAnalysisHvdcParameterInfos {
+
+    private SensitivityAnalysisInputData.SensitivityType sensitivityType;
+
+    List<FilterEquipments> monitoredBranches;
```

Comment:
Those 3 fields are present in SensitivityAnalysisInjectionsParameterInfos, SensitivityAnalysisHvdcParameterInfos, SensitivityAnalysisPtsParameterInfos, SensitivityAnalysisNodesParameterInfos. 
You could do a class of it. 

And same for entities. You create several tables that have the same structure. There might be something simpler to be done